### DISCUSSION TITLE
starship update fail in the windows11

### DISCUSSION THREAD

**Discussion by QingyaoLin** [starship/starship#3707] [Q&A] [answered]
I used cargo Install-update-a for the update.
Here are the update tips and failure tips.

Package             Installed  Latest   Needs update
starship            v1.3.0     v1.4.0   Yes
.......
Updating starship
    Updating crates.io index
  Installing starship v1.4.0
   Compiling autocfg v1.1.0
   Compiling version_check v0.9.4
.......
error: failed to run custom build command for `starship v1.4.0`

Caused by:
  process didn't exit successfully: `C:\Users\26504\AppData\Local\Temp\cargo-installReEnDA\release\build\starship-2184158819d86872\build-script-build` (exit code: 1)
  --- stdout
  could not find repository from 'C:\Users\26504\.cargo\registry\src\github.com-1ecc6299db9ec823\starship-1.4.0'; class=Repository (6); code=NotFound (-3)
  cargo:rerun-if-env-changed=SOURCE_DATE_EPOCH
  cargo:rerun-if-env-changed=SHORT_COMMIT
  cargo:rerun-if-env-changed=GIT_CLEAN
  cargo:rerun-if-env-changed=PKG_DESCRIPTION
  cargo:rerun-if-env-changed=PKG_VERSION_PRE
  cargo:rerun-if-env-changed=BUILD_TIME
  cargo:rerun-if-env-changed=COMMIT_HASH
  cargo:rerun-if-env-changed=BUILD_TIME_2822
  cargo:rerun-if-env-changed=COMMIT_AUTHOR
  cargo:rerun-if-env-changed=BUILD_TARGET_ARCH
  cargo:rerun-if-env-changed=PKG_VERSION_PATCH
  cargo:rerun-if-env-changed=BUILD_OS
  cargo:rerun-if-env-changed=CARGO_TREE
  cargo:rerun-if-env-changed=COMMIT_DATE_2822
  cargo:rerun-if-env-changed=COMMIT_DATE
  cargo:rerun-if-env-changed=BUILD_TARGET
  cargo:rerun-if-env-changed=BUILD_TIME_3339
  cargo:rerun-if-env-changed=BRANCH
  cargo:rerun-if-env-changed=TAG
  cargo:rerun-if-env-changed=COMMIT_DATE_3339
  cargo:rerun-if-env-changed=CARGO_VERSION
  cargo:rerun-if-env-changed=PROJECT_NAME
  cargo:rerun-if-env-changed=PKG_VERSION_MAJOR
  cargo:rerun-if-env-changed=PKG_VERSION_MINOR
  cargo:rerun-if-env-changed=RUST_CHANNEL
  cargo:rerun-if-env-changed=COMMIT_EMAIL
  cargo:rerun-if-env-changed=RUST_VERSION
  cargo:rerun-if-env-changed=GIT_STATUS_FILE
  cargo:rerun-if-env-changed=PKG_VERSION
  cargo:rerun-if-env-changed=BUILD_RUST_CHANNEL
  package.metadata does not exist
  "C:\\Program Files (x86)\\Windows Kits\\10\\bin\\10.0.19041.0\\x64\\rc.exe"
  Selected RC path: 'C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x64\rc.exe'
  RC Output:
  Microsoft (R) Windows (R) Resource Compiler Version 10.0.10011.16384
  Copyright (C) Microsoft Corporation.  All rights reserved.

  C:\Users\26504\AppData\Local\Temp\cargo-installReEnDA\release\build\starship-15f47efae5847737\out\resource.rc(25) : error RC2135 : file not found: starship.exe.manifest

  ------
  RC Error:

  ------

  --- stderr
  fatal: not a git repository (or any of the parent directories): .git
  fatal: not a git repository (or any of the parent directories): .git
  Error: Custom { kind: Other, error: "Could not compile resource file" }
warning: build failed, waiting for other jobs to finish...
error: failed to compile `starship v1.4.0`, intermediate artifacts can be found at `C:\Users\26504\AppData\Local\Temp\cargo-installReEnDA`

Caused by:
  build failed

Updated 0 packages.
Failed to update starship.

**Comment by chipbuster** [comment, marked as answer]
Hopefully this should have been fixed by the v1.4.2 release.

**Reply by QingyaoLin** [reply]
Yeah,. Fixed in version 1.4.2
